src/main/java/org/takes/http/BkBasic.java

```diff
@@ -54,10 +54,35 @@
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle IndentationCheck (500 lines)
  */
+@SuppressWarnings("PMD.LongVariable")
 @EqualsAndHashCode(of = "take")
 public final class BkBasic implements Back {
 
     /**
+     * The http header "X-Takes-LocalAddress".
+     */
+    public static final String HEADER_TAKES_LOCAL_ADDRESS =
```

Comment:
@yegor256 Well, the ticket require them as public to be used also from tests. So it remains only to encapsulate them as Header instances with toString() method.